scripts/code_size_compare.py

```diff
@@ -29,18 +29,84 @@
 import os
 import subprocess
 import sys
+from enum import Enum
 
 from mbedtls_dev import build_tree
 
+class SupportedArch(Enum):
+    """Supported architecture for code size measurement."""
+    AARCH64 = 'aarch64'
+    AARCH32 = 'aarch32'
```

Comment:
As before, AArch32 is an execution state of Armv8-A that is roughly equivalent to Armv7-A, for A-class processors.

This setting compiles for M-class architectures, not A-class, so we should call it `armv8-m` or similar